### PR TITLE
Add mofo-style, eslint, update package.json fields

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

**Comment by alanmoo** [comment]
Oops, trying to use ES6 currently doesn't work because there's no Babel, so eslint flags unfixable errors.

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=759670
+	// for the documentation about the jsconfig.json format
```
Comment (mmmavis) [inline jsconfig.json]: interesting that Visual Studio allows comments in `.json` 😮

File: package.json [hunk 2]
```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0",
   "description": "Mozfest site",
   "author": "Mozilla Foundation",
+  "repository": "mozilla/mozillafestival.org",
+  "license": "MPL-2.0",
```
Comment (mmmavis) [inline package.json]: Just realized we don't have a `LICENSE` file on this repo. Do you mind adding it in this PR @alanmoo ?

Comment (alanmoo) [inline package.json]: Done

File: jsconfig.json (hunk 1 above)
Comment (alanmoo) [inline jsconfig.json]: Yeah, this was a generated file by them too.

**Comment by mmmavis** [comment]
Worked as a charm ✨ R+

**Pull request merged**